src/main/java/wooteco/subway/domain/fare/AgeDiscountPolicy.java

```diff
@@ -0,0 +1,50 @@
+package wooteco.subway.domain.fare;
+
+import java.util.Arrays;
+
+public enum AgeDiscountPolicy {
+
+    BABY(0, Constants.CHILD_MIN_AGE, 0, 1.00),
+    CHILD(Constants.CHILD_MIN_AGE, Constants.ADOLESCENT_MIN_AGE, 350, 0.50),
+    ADOLESCENT(Constants.ADOLESCENT_MIN_AGE, Constants.ADULT_MIN_AGE, 350, 0.20),
+    ADULT(Constants.ADULT_MIN_AGE, Constants.ELDERLY_MIN_AGE, 0, 0.00),
+    ELDERLY(Constants.ELDERLY_MIN_AGE, 150, 0, 1.00),
+    ;
+
+    private static final String INVALID_AGE_RANGE_EXCEPTION = "0과 150 사이의 연령만 입력가능합니다.";
```

Comment:
```suggestion
    private static final String INVALID_AGE_RANGE_EXCEPTION = Constants.MIN_AGE + "과 " + Constants.MAX_AGE + " 사이의 연령만 입력가능합니다.";
```

상수로 관리하면 요구사항이 바꼈을 때 메시지도 놓치지 않고 변경할 수 있겠네요

Reply:
<img width="580" alt="image" src="https://user-images.githubusercontent.com/73531614/170489330-cd4a6d32-450c-498f-b088-fed083f13a75.png">